clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -242,7 +242,7 @@ Flags<[WrapperOnlyOption]>,  HelpText<"Embed LLVM IR for runtime kernel fusion">
 
 def sycl_dump_device_code_EQ : Joined<["--", "-"], "sycl-dump-device-code=">,
   Flags<[WrapperOnlyOption]>,
-  HelpText<"Path to the folder where the tool dumps SPIR-V device code. Other formats aren't dumped.">;
+  HelpText<"Directory to dump offloading images to.">;
```

Comment:
This option mirrors the user facing option 'sycl-dump-device-code' which was introduced for the old offloading model here: https://github.com/intel/llvm/pull/11132. In that PR, I see that we use this only for dumping SPIR-V code. If the idea is to extend this option to cover all possible device outputs, may be we should do this for the old offloading model as well?

Thanks

Reply:
I'm not sure if there is any interest in supporting additional formats for "old offloading model". 
The old offloading model is supposed to be replaced with the new offloading model soon. Right?

FYI: non-SPIR-V targets seems to be happy with -fsycl-device-only output. See https://github.com/intel/llvm/pull/17390.

If we have customers for this use case, please, file a feature request to handle it separately.